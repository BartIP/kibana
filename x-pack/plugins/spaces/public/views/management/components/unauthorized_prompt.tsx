/*
 * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
 * or more contributor license agreements. Licensed under the Elastic License;
 * you may not use this file except in compliance with the Elastic License.
 */

import { EuiEmptyPrompt } from '@elastic/eui';
<<<<<<< HEAD
=======
import { FormattedMessage } from '@kbn/i18n/react';
>>>>>>> ff49a1c6742d67fa5daed569ff3bb269783f6bd1
import React from 'react';

export const UnauthorizedPrompt = () => (
  <EuiEmptyPrompt
    iconType="spacesApp"
    iconColor={'danger'}
<<<<<<< HEAD
    title={<h2>Permission denied</h2>}
    body={
      <p data-test-subj="permissionDeniedMessage">You do not have permission to manage spaces.</p>
=======
    title={
      <h2>
        <FormattedMessage
          id="xpack.spaces.management.unauthorizedPrompt.permissionDeniedTitle"
          defaultMessage="Permission denied"
        />
      </h2>
    }
    body={
      <p data-test-subj="permissionDeniedMessage">
        <FormattedMessage
          id="xpack.spaces.management.unauthorizedPrompt.permissionDeniedDescription"
          defaultMessage="You do not have permission to manage spaces."
        />
      </p>
>>>>>>> ff49a1c6742d67fa5daed569ff3bb269783f6bd1
    }
  />
);
