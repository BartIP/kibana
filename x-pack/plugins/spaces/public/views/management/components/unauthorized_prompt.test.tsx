/*
 * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
 * or more contributor license agreements. Licensed under the Elastic License;
 * you may not use this file except in compliance with the Elastic License.
 */
<<<<<<< HEAD
import { shallow } from 'enzyme';
import React from 'react';
=======
import React from 'react';
import { shallowWithIntl } from 'test_utils/enzyme_helpers';
>>>>>>> ff49a1c6742d67fa5daed569ff3bb269783f6bd1
import { UnauthorizedPrompt } from './unauthorized_prompt';

describe('UnauthorizedPrompt', () => {
  it('renders as expected', () => {
<<<<<<< HEAD
    expect(shallow(<UnauthorizedPrompt />)).toMatchSnapshot();
=======
    expect(shallowWithIntl(<UnauthorizedPrompt />)).toMatchSnapshot();
>>>>>>> ff49a1c6742d67fa5daed569ff3bb269783f6bd1
  });
});
