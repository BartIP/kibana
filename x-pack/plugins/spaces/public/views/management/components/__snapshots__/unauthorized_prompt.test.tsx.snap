// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnauthorizedPrompt renders as expected 1`] = `
<EuiEmptyPrompt
  body={
    <p
      data-test-subj="permissionDeniedMessage"
    >
<<<<<<< HEAD
      You do not have permission to manage spaces.
=======
      <FormattedMessage
        defaultMessage="You do not have permission to manage spaces."
        id="xpack.spaces.management.unauthorizedPrompt.permissionDeniedDescription"
        values={Object {}}
      />
>>>>>>> ff49a1c6742d67fa5daed569ff3bb269783f6bd1
    </p>
  }
  iconColor="danger"
  iconType="spacesApp"
  title={
    <h2>
<<<<<<< HEAD
      Permission denied
=======
      <FormattedMessage
        defaultMessage="Permission denied"
        id="xpack.spaces.management.unauthorizedPrompt.permissionDeniedTitle"
        values={Object {}}
      />
>>>>>>> ff49a1c6742d67fa5daed569ff3bb269783f6bd1
    </h2>
  }
/>
`;
